parallel-consumer-python/pyallel_consumer/pom.xml

```
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.confluent.parallelconsumer</groupId>
+    <artifactId>python</artifactId>
```

lets stick with the rest of the convention for now
```suggestion
    <artifactId>parallel-consumer-python</artifactId>
```